pkg/apis/hive/v1/validating-webhooks/clusterimageset_validating_admission_hook.go

```diff
@@ -148,8 +148,8 @@ func (a *ClusterImageSetValidatingAdmissionHook) validateCreate(admissionSpec *a
 	// Add the new data to the contextLogger
 	contextLogger.Data["object.Name"] = newObject.Name
 
-	if newObject.Spec.InstallerImage == nil && newObject.Spec.ReleaseImage == nil {
-		message := "Failed validation: you must specify either an installer image or a release image"
+	if newObject.Spec.ReleaseImage == "" {
+		message := "Failed validation: you must specify either a release image"
```

Comment:
"must specify either a release image"
either a release image, or what else are the options? or should this simply be "must specify a release image"?

Reply:
It should be "must specify a release image": There is no other option.